__tests__/app.test.js

```diff
@@ -149,6 +149,24 @@ describe("GET /api/articles", () => {
         expect(body.articles).toBeSortedBy("votes", { ascending: true });
       });
   });
+  test("200: filters articles by topic when topic query is provided", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((article) => {
+          expect(article.topic).toBe("cats");
+        });
```

Comment:
Really nice test just checking the topic to be "cats" on all the articles, that's all we're adding here so that's all we should check 🥇